### ISSUE TITLE
Loading prmtop file error

### ISSUE THREAD

**Issue by tonyyzy** [arose/mdsrv#43]
When importing a prmtop file, generated as part of the Amber toolkit, the protein's structure is not loaded and only a single atom is displayed as shown below. It's the same when the trajectory file is imported as well and only the first frame contains this single atom.

![image](https://user-images.githubusercontent.com/38984697/62329317-7cf0ab80-b4ad-11e9-8758-44bd006e62b5.png)

@j0kaso

**Comment by j0kaso** [comment]
Yes, this is normal for structure files which do by themself not contain any atom coordinates. If you generate a js/ngl script to load them, you can set the first trame which will be shown to 1.

Please note: frame=0 are the coordinates from the structure file and only from 1 the trajectory coordinates are visualized

**Comment by tonyyzy** [comment]
@j0kaso 
The problem I'm facing is that even after loading the trajectory file (NetCDF format), still no structure is displayed.

So after loading prmtop with .nc file, the `-1` frame shows the single atom, as shown in the previous image and all other frames are empty.
